README.md

```diff
@@ -50,6 +57,7 @@ Where:
 * `APPLICATION_ID` is the identifier generated by Yoti Dashboard when you create your app.
 * `path/to/your-application-pem-file.pem` is the path to the pem file your browser generates for you, when you create your app on Yoti Dashboard.
 
+Example reference: [node/example/simple-login/index.js.](https://github.com/getyoti/node/blob/master/example/simple-login/index.js)
```

Comment:
As this will also be on NPM, I would suggest `[Example reference](https://github.com/getyoti/node/blob/master/example/simple-login/index.js)`